eogrow/utils/ray.py

```diff
@@ -0,0 +1,36 @@
+"""
+Modules with Ray-related utilities
+"""
+import logging
+
+import ray
+
+from .types import BoolOrAuto
+
+LOGGER = logging.getLogger(__name__)
+
+
+def connect_to_ray(use_ray: BoolOrAuto = "auto") -> bool:
```

Comment:
not sure about the choice of name, especially `connect_to_ray(use_ray=False)` sounds a bit odd. Don't have suggestions so far.

Reply:
Hm, I also can't find a better name for this. :thinking: 

Atm, I can think of `resolve_ray_connection`, `setup_ray`, `ensure_ray_requirement`. :thinking:

Reply:
Renamed to `handle_ray_connection`. I'm not 100% happy but it's better than before.

Reply:
I also like it better